py/py3/sbsearch.py

```diff
@@ -39,7 +39,7 @@
 if len(sys.argv) >= 5:
     filter_ = SearchFilter(sys.argv[4])
 else:
-    filter_ = SearchFilter("loopless")
+    filter_ = SearchFilter("LOOPLESS")
 
```

Comment:
this should just be `filter_ = SearchFilter.LOOPLESS`

Reply:
Not sure how that happened. I'll make sure that's fixed in the next PR.